cmd/podman/common/create.go

```diff
@@ -327,13 +327,10 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 		)
 		_ = cmd.RegisterFlagCompletionFunc(ipcFlagName, AutocompleteNamespace)
 
-		kernelMemoryFlagName := "kernel-memory"
-		createFlags.StringVar(
-			&cf.KernelMemory,
-			kernelMemoryFlagName, "",
+		createFlags.String(
+			"kernel-memory", "",
 			"Kernel memory limit "+sizeWithUnitFormat,
```

Comment:
Shouldm't we say "Deprecated" here?

Reply:
Sure as a documentation thing, but this option is hidden anyways.